include/z64pause.h

```diff
@@ -184,7 +186,7 @@ typedef struct PauseContext {
     /* 0x0262 */ s16 promptChoice; // save/continue choice: 0 = yes; 4 = no
     /* 0x0264 */ s16 ocarinaSongIdx;
     /* 0x0266 */ u8 worldMapPoints[20]; // indices: `WorldMapPoint` enum, values: `WorldMapPointState` enum
-    /* 0x027A */ u8 tradeQuestLocation;
+    /* 0x027A */ u8 tradeQuestMarker;
```

Comment:
maybe comment that this takes `WORLD_MAP_POINT_` or is `TRADE_QUEST_MARKER_NONE`?